patches/openssl/0002-Add-configuration-file-for-Managarm-targets.patch

```diff
@@ -0,0 +1,68 @@
+From d3811a0350dcc9d53d919b5c7b69b92173f1f48e Mon Sep 17 00:00:00 2001
+From: Dennis Bonke <admin@dennisbonke.com>
+Date: Tue, 1 Jun 2021 22:03:10 +0200
+Subject: [PATCH 2/2] Add configuration file for Managarm targets. Initial work
+ done by Qookie
+
+Signed-off-by: Dennis Bonke <admin@dennisbonke.com>
+---
+ Configurations/10-main.conf | 43 +++++++++++++++++++++++++++++++++++++
+ 1 file changed, 43 insertions(+)
+
+diff --git a/Configurations/10-main.conf b/Configurations/10-main.conf
+index 8dc3e858df..4b4f684578 100644
+--- a/Configurations/10-main.conf
++++ b/Configurations/10-main.conf
+@@ -911,6 +911,49 @@ my %targets = (
+         ranlib           => "true",
+     },
+ 
++#### Managarm
++    "managarm-generic64" => {
++        inherit_from     => [ "BASE_unix" ],
++        CFLAGS           => picker(default   => "-Wall",
++                         debug       => "-O0 -g",
++                         release     => "-O3"),
++        CXXFLAGS         => picker(default   => "-Wall",
++                         debug       => "-O0 -g",
++                         release     => "-O3"),
++        cxxflags         => add("-std=c++11"),
++        lib_cppflags     => combine("-DOPENSSL_USE_NODELETE", "-DL_ENDIAN"),
++        bn_ops           => "SIXTY_FOUR_BIT_LONG RC4_CHAR",
```

Comment:
Does Linux use this for both x86_64 and aarch64?

Reply:
Seems like it (assuming you're talking about the `bn_ops` thing.